cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -51,6 +55,31 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
       );
     });
 
+    it('if a legacy MAP it should render a link using an a tag and href, rather than a plain text <link>', () => {
```

Comment:
Do you want to reword this to `legacy MAP should render a link using an <a> tag with href rather than a plain text <link>`

Reply:
👍